src/status_im/utils/contacts.cljs

```diff
@@ -1,7 +1,8 @@
 (ns status-im.utils.contacts
```

Comment:
I wonder if this namespace can be merged easily into contacts.core without circular dependencies. but that's not really the point of this PR so it's ok for me. This namespace should disappear in the future though

Reply:
It is probably just a matter of convention but if there will be more of functions like
find-contact-by-address, find-contact-by-name etc. then maybe I would think about creating separate contacts.queries namespace, because for me utils are mostly created in case when you can't change source code of some namespace/class in library that you use and you want to extend it (like for example often is the case with String class in Java when classes like StringUtils are created), but again this is a matter of convention and I did not want to change too much because I don't have enough knowledge about whole project.

Reply:
yes the convention for that kind of function is to put it in a db namespace